php/ubiquity/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"require" : {
+		"php" : "^7.1",
+		"phpmv/ubiquity" : "dev-master"
+	},
+	"require-dev": {
+		"mindplay/annotations": "^1.3",
+		"phpmv/ubiquity-dev": "dev-master",
+		"phpmv/ubiquity-devtools": "dev-master"
```

Comment:
should it not be used for **production** also ? (despite the name)

Reply:
no, this is only for dev

Reply:
I mean **devtools**, there `here` to do some stuff like warming up caches no ?
Some kind of things we **MUST** run on **production**, as an example, I had to add trhe console manager on `symfony` to warmup the cache on **production**
https://github.com/the-benchmarker/web-frameworks/blob/master/php/symfony/config.yaml#L15

Reply:
**devtools** allow the generation of the cache for production, indeed. But you don't leave them in place in production, for performance reasons.
You just have one single command to do that:
`Ubiquity bootstrap prod`
This should also be the case for the Symfony console.

Reply:
I'm not a  `php` dev, but for example, in `rails` we generally leave the file `bin/rails` in **production**.
having those **binstubs** (`bin/rails` is one of them) are used in **production** to migrate the database (as an example) so having them in **production** is required.
It seems that `ubiquity/devtools` is the same, no ?

Reply:
No, because **devtools** can be installed globally (with `composer global require phpmv/ubiquity-devtools`).
This avoids **dev** parts in a project in production.